src/Glottis20Mint.sol

```diff
@@ -94,26 +94,34 @@ contract Glottis20Mint is ReentrancyGuard {
         uint256 liquidityEth = ethLiquidity.rawSub(protocolEthFee).rawSub(creatorEthFee).rawSub(callerEthFee);
 
         // Transfer fees
+        // fico23:
+        // why are we forcing? why not just SafeTransferLib.safeTransferETH?
+        // force will call selfdestruct - so im not sure how it works with 3 in a row
+        // even if it does work, its better to use safe transfer
```

Comment:
from soLady safeTransferLib.sol:

/// @dev Note:
/// - For ETH transfers, please use forceSafeTransferETH for DoS protection.

not sure why/how it works, just followed instructions as I noticed it while browsing the source.